src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+.block_item_1 {
+  height: 100px;
+  width: 100%;
+  background-color: red;
+}
+
+.block_item_2 {
+  height: 100px;
+  width: 100%;
+  background-color: green;
+}
+
+.block_item_3 {
+  height: 100px;
+  width: 100%;
+  background-color: blue;
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.